### PR TITLE
Bug 1574482 - Bad tokenization for preproc directives containing tabs

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: tools/src/tokenize.rs [hunk 1]
```diff
@@ -291,7 +291,7 @@ pub fn tokenize_c_like(string: &str, spec: &LanguageSpec) -> Vec<Token> {
             tokens.push(Token {start: start, end: nl, kind: TokenKind::Comment});
             tokens.push(Token {start: nl, end: peek_pos(), kind: TokenKind::Newline});
         } else if ch == '#' && spec.c_preprocessor {
-            while peek_char() == ' ' {
+            while peek_char() == ' ' || peek_char() == '\t' {
```
Comment (emilio) [inline tools/src/tokenize.rs]: Should these also handle other kinds of whitespace?

Comment (calixteman) [reply tools/src/tokenize.rs]: Good question.
According to this line: https://github.com/mozsearch/mozsearch/blob/master/tools/src/tokenize.rs#L27
whitespace could be a "\r" but could we really have such a char after a "#" ?

Comment (emilio) [reply tools/src/tokenize.rs]: Probably. Anyhow this can probably be a follow-up. Let's get this merged.

**Comment by emilio** [comment]
Thanks for the patch!

**Pull request merged**